snapcraft/commands/lifecycle.py

```diff
@@ -89,6 +95,16 @@ def fill_parser(self, parser: "argparse.ArgumentParser") -> None:
             nargs="*",
             help="Optional list of parts to process",
         )
+        parser.add_argument(
```

Comment:
these 2 (should probably be a mutually exclusive group)

Reply:
Added to group.